lib/clearinghouse/build.gradle.kts

```
@@ -0,0 +1,34 @@
+plugins {
+    id ("java")
+    id ("maven-publish")
+    id("io.freefair.lombok") version "8.4"
+}
```

The formatting plugin should be added here, else spotless won't check files in this module.
```suggestion
    id("formatting-conventions")
}
```